File: app/src/main/java/com/osapps/capitalslearnproject/fragmentactivity/frags/detail/presentation/DetailFragmentPresenter.java
package com.osapps.capitalslearnproject.fragmentactivity.frags.detail.presentation;

import com.osapps.capitalslearnproject.fragmentactivity.frags.detail.view.DetailFragmentView;

import javax.inject.Inject;

/**
 * Created by mertsimsek on 02/06/2017.
 */

public class DetailFragmentPresenter {

    DetailFragmentView detailFragmentView;

    @Inject
    public DetailFragmentPresenter(DetailFragmentView detailFragmentView) {
        this.detailFragmentView = detailFragmentView;
    }

    public void viewLoaded() {
        detailFragmentView.onDetailFragmentLoaded();
    }
}

File: app/src/main/java/com/osapps/capitalslearnproject/fragmentactivity/frags/detail/presentation/DetailFragmentPresenter.kt
package com.osapps.capitalslearnproject.fragmentactivity.frags.detail.presentation

import com.osapps.capitalslearnproject.fragmentactivity.frags.detail.view.DetailFragmentView

import javax.inject.Inject

/**
 * Created by osApps on 02/06/2017.
 */

class DetailFragmentPresenter @Inject
constructor(private var detailFragmentView: DetailFragmentView) {

    fun viewLoaded() {
        detailFragmentView.onDetailFragmentLoaded()
    }
}
